### PR TITLE
Proposal to include Sphinx guides to the PPR project. 

### LINKED ISSUE #50: Incorporate Sphinx Guides to the PPR project.
As part of the contributions of the Dataverse team we would like to incorporate the guides framework that we use, Sphinx has support for MD and creates guides with the help of Python. 

We want to document part of the knowledge that @abujeda is sharing with us and make the README.md file shorter since most of the documentation is now there.

### PR DESCRIPTION
**Pull request by jp-tosca**
**What this PR does / why we need it**:
As part of the contributions of the Dataverse team we would like to incorporate the guides framework that we use, Sphinx has support for MD and creates guides with the help of Python.

We want to document part of the knowledge that @abujeda is sharing with us and make the README.md file shorter since most of the documentation is now there.

**Which issue(s) this PR closes**:

Closes #50 

**Special notes for your reviewer**:

All the content is set up on the docs/Sphinx-guides folder, if you think there is a better place please let me know. 

**Suggestions on how to test this**:

Instructions on docs/Sphinx-guides/HOWTO-SPHINX-INSTALL.txt

### CODE REVIEW

**Comment by abujeda** [comment]
What do you think about using a Sphinx Docker image to compile the guide: https://hub.docker.com/r/sphinxdoc/sphinx?
This is to avoid having to install locally Python, Sphinx, and other libs

```bash
docker run --rm -it -v $PWD:/docs sphinxdoc/sphinx:7.3.7 /bin/bash
pip install -r requirements.txt
make html
```

File: docs/Sphinx-guides/HOWTO-SPHINX-INSTALL.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+And this is how you build: 
+
+make html
+
+in this directory will generate html files, and leave them in the build subdirectory here. 
+To have it build the html files AND deploy them under your glassfish environment, type
+
+make htmldeploy
+
+For this the Makefile assumes that your glassfish is in /usr/local/glassfish4.
```
Comment (abujeda) [inline docs/Sphinx-guides/HOWTO-SPHINX-INSTALL.txt:32]: Is this correct for default Sphinx installations?
`htmldeploy` will try to copy the files into `/usr/local/glassfish4`

Should we simply update to say that to deploy the HTML guide into an specific folder use:
make htmldeploy DOCROOT=/folder/to/deploy

Comment (jp-tosca) [reply docs/Sphinx-guides/HOWTO-SPHINX-INSTALL.txt]: I also added a document with instructions and commands to build with docker as suggested 👍🏼

**Comment by abujeda** [comment]
LGTM - Merging

**Pull request merged**